client/js/libs/handlebars/friendlydate.js

```diff
@@ -4,10 +4,10 @@ const moment = require("moment");
 
 module.exports = function(time) {
 	// See http://momentjs.com/docs/#/displaying/calendar-time/
-	return moment(new Date(time)).calendar(null, {
+	return moment(time).calendar(null, {
 		sameDay: "[Today]",
 		lastDay: "[Yesterday]",
-		lastWeek: "L", // Locale
-		sameElse: "L"
+		lastWeek: "MMMM DD, YYYY",
+		sameElse: "MMMM DD, YYYY"
```

Comment:
I don't think this is a win. That way of displaying dates is only common in very few countries, so relying on locales is safer (unless I'm missing something).

I think this should be dealt with when internationalizing the client, and left alone until then.